### DISCUSSION TITLE
Hidden Front-Matter option for category use

### DISCUSSION THREAD

**Discussion by TheOrdinaryWow** [importantimport/urara#37] [Ideas]
突然想到一个想法，个人认为必要性有点大。

假设我想新建一个子分类「Alpha」，并将文章按如下结构放置：
```
...
Alpha/Article-1
Alpha/Article-2
Alpha/Article-3
```

但同时，我只希望 Alpha 作为一个分类项目，而不是一篇文章。而这个操作在 Urara 中是不允许的。

因此，若可以在「Alpha」的 Front-Matter 添加参数使得 Urara 识别这是一个分类而不是一篇文章，或直接隐藏这个页面不在 Index 中显示，只能通过输入 URL 访问，则可以解决这个问题。如：

若允许将这个页面设置为分类，则该页面不显示在 Index 中，访问该 URL 则会返回该分类下所有文章（类似于 `?tags=$tags` 页面）。
```
---
title: Alpha
category: true
---
```

若允许将这个页面设置为隐藏，也不会在 Index 中显示，只能通过访问该 URL 打开。

```
---
title: Alpha
no-index: true
---
......
```

**Comment by kwaa** [comment]
前者目前还无法实现，不过后者可以通过
```yaml
flags:
- unlisted
```
来做到。

**Reply by TheOrdinaryWow** [reply]
感谢！
